packages/filezilla.rb

```diff
@@ -8,19 +8,6 @@ class Filezilla < Package
   source_url 'https://download.filezilla-project.org/client/FileZilla_3.51.0_src.tar.bz2'
```

Comment:
`version '3.51.0-1'`?

Reply:
I can't find this version in `https://download.filezilla-project.org/client/`

Reply:
It's not a real version.  I just gave you an example of how you can bump.